app/models/base.py

```diff
@@ -0,0 +1,61 @@
+from datetime import datetime, timezone
+from typing import Optional
+
+from pydantic import (
+    BaseModel,
+    AnyUrl,
+    Field,
+    UUID4,
+)
+
+
+class LinkSettingsModel(
+    BaseModel,
+):
+    title: Optional[str] = Field("")
+    description: Optional[str] = Field("")
+    archived: bool = Field(False)
```

Comment:
Hm, is there a reason why use always use `Field` as a default value? It should work exactly the same way:

```suggestion
    title: Optional[str] = ""
    description: Optional[str] = ""
    archived: bool = False
```

Reply:
Probably, you either want to make `None` a default value:

```suggestion
    title: Optional[str] = None
    description: Optional[str] = None
    archived: bool = False
```

Or remove `Optional`:

```suggestion
    title: str = ""
    description: str = ""
    archived: bool = False
```

Libo krestik snimi, libo trusy naden.

Reply:
I use `Filed` as a default value because I plan to extend the description for this field of the model